source/_integrations/yeelight.markdown

```diff
@@ -16,33 +18,60 @@ The `yeelight` integration allows you to control your Yeelight Wi-Fi bulbs with
 There is currently support for the following device types within Home Assistant:
 
 - **Light** - The Yeelight platform for supporting lights.
-- **Sensor** - The Yeelight platform for supporting sensors. Currently only nightlight mode sensor, for ceiling lights.
+- **Binary Sensor** - The Yeelight platform for supporting binary sensors. Currently only nightlight mode sensor, for ceiling lights.
 
-## Example configuration (Automatic)
+## Configuration
 
-After the lights are connected to the Wi-Fi network and have been detected in Home Assistant, the discovered names will be shown in the `Light` section of the `Overview` view. Add the following lines to your `customize.yaml` file:
+To set up this integration, goto Integrations udner Configuration, find Yeelight in the + menu. You can choose "Discovery" or "Manual". To configure the discovered or manually set up devices, use the "Options" button in the integration card. **Note: If you have discovery set up, do not manually set up devices that are discoverable. Otherwise there will be conflicts.**
```

Comment:
```suggestion
To set up this integration, go to Integrations under Configuration and find Yeelight in the + menu. You can choose "Discovery" or "Manual". To configure the discovered or manually set up devices, use the "Options" button in the integration card. **Note: If you have discovery set up, do not manually set up devices that are discoverable. Otherwise there will be conflicts.**
```